csharp-basics/exercises/ClassesAndObjects/DateTest/VideoStoreTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using VideoStore;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VideoStore.Tests
+{
+    [TestClass]
+    public class VideoStoreTests
+    {
+        [TestMethod]
+        public void ListInventoryTest()
+        {
+            // Arrange
+            VideoStore videoStore = new VideoStore();
+
+            List<string> listOfTitles = new List<string>();
+
+            listOfTitles.Add("The Matrix");
+            listOfTitles.Add("Godfather II");
+            listOfTitles.Add("Star Wars Episode IV: A New Hope");
+            
+            foreach (var title in listOfTitles)
+            {
+                videoStore.AddVideo(title);
```

Comment:
Kādu metodi tu šeit testē? Testā nav jāliek loopi, visam ir jābū skaidri definētam, bez loopiem vai citām darbībām, izņemot act daļu, kur testē konkrēto metodi

Reply:
nav labots